src/config/model.c

```diff
@@ -672,13 +687,18 @@ int assign_int(void* ptr, const struct struct_map *map, int map_size)
         if(assign_int(&Model, _secradio, MAPSIZE(_secradio)))
             return 1;
         if (MATCH_KEY(RADIO_TX_POWER)) {
-            for (i = 0; i < NUM_STR_ELEMS(RADIO_TX_POWER_VAL); i++) {
-                if (MATCH_VALUE(RADIO_TX_POWER_VAL[i])) {
+            if (m->radio == TX_MODULE_LAST) {
+                m->tx_power = TXPOWER_150mW;
+                return 1;
+            }
+            for (i = 0; i < RADIO_TX_POWER_COUNT[m->radio]; i++) {
+                if (MATCH_VALUE(RADIO_TX_POWER_VAL[m->radio][i])) {
                     m->tx_power = i;
                     return 1;
                 }
             }
             printf("Unknown Tx power: %s\n", value);
+            m->tx_power = RADIO_TX_POWER_COUNT[m->radio]-1; // default to radio maximum
```

Comment:
Lint Error:
```
701:  At least two spaces is best between code and comments  
```

Reply:
There's other single space instances in the same file.  Are only changes going through lint?  And only flagged instances should be changed?